libecho/src/echo.cpp

```diff
@@ -0,0 +1,20 @@
+#include "echo.h"
+#include <iostream>
+
+void echo::send(uint8_t *buffer, size_t len) {
+    std::cout << "Sending: ";
+    for (size_t i = 0; i < len; i++) {
+        std::cout << (char)buffer[i];
```

Comment:
undefined behavior dla wartości spoza zakresu typu char

Reply:
Samo rzutowanie to raczej nie UB? Ale chyba jest niepotrzebne, zrobiłem żeby printowało chara a nie liczbę, co jest chyba zbędne prz uint8_t.

Reply:
Wywaliłem